remix/app/components/FormInput/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import FormInput from './';
+
+jest.mock('remix-validated-form', () => ({
+  useField: jest.fn().mockReturnValue({
+    error: 'Test error',
+    getInputProps: jest.fn().mockReturnValue({}),
+  }),
+}));
+
+describe('FormInput', () => {
+  describe('givens FormInput attribute', () => {
```

Comment:
You keep making this typo and here it might be about assigning valid props:

```suggestion
  describe('given valid props', () => {
```

Reply:
😅 yes, I will be more careful about it.

Reply:
Improved here: [0af22fb](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/0af22fb487b5618b545e2edf90f9b9e0fe94da67)